pkg/kubelet/util/boottime_util_linux.go

```diff
@@ -21,17 +21,26 @@ package util
 
 import (
 	"fmt"
+	"os"
+	"strconv"
+	"strings"
 	"time"
-
-	"golang.org/x/sys/unix"
 )
 
-// GetBootTime returns the time at which the machine was started, truncated to the nearest second
+// GetBootTime returns the time at which the machine was started, truncated to the nearest second.
 func GetBootTime() (time.Time, error) {
 	currentTime := time.Now()
-	var info unix.Sysinfo_t
-	if err := unix.Sysinfo(&info); err != nil {
+	raw, err := os.ReadFile("/proc/uptime")
```

Comment:
Let's fall back to `unix.Sysinfo` if we can't get the info from the file for some reason or other.